packages/common/src/stream.ts

```diff
@@ -90,10 +90,10 @@ export interface StreamNext {
   metadata?: StreamMetadata
 }
 
-export enum CommitType {
-  GENESIS,
-  SIGNED,
-  ANCHOR,
+export enum EventType {
```

Comment:
can we add comments that these used to be called genesis, signed, and anchor, and may still be referred to this way by some other code and comments?  Just to avoid confusion for folks who see types like `GenesisHeader` and aren't sure if genesis and init mean the same thing (they do).

And also maybe add some TODO comments over the other types in this file like `GenesisHeader` to rename them to be aligned with the new naming conventions.